README.md

```diff
@@ -1 +1,69 @@
+
+
+## Built With
+
+- HTML, CSS
+- No Frameworks
+- Technologies Used- HTML, CSS, GIT, Linters (GitHub Actions, Lighthouse, Webhint, Stylelint)
+
+## Getting Started
+<ol>
+   <li>**Create a new GitHub repository.</li>
+   <li>**Clone the repository to the local env through commandline/terminal or code editors like Visual Studio Code.</li>
+   <li>**Create a new feature branch.</li>
+   <li>**Create .gitignore file.</li>
+   <li>**Set up GitHub Actions and linters in the local env.</li>
+   <li>**Add the main files and folders to the main branch.</li>
+   <li>** Add the following files in separate commits-</li>
+     <ol>
+       <li>A .html file for the skeletal makeup of the webpage.</li>
+       <li>A .css file for styling the webpage.</li>
+     </ol>
+   <li>**Open a pull request.</li>
+   <li>**Submit the project for the code review.</li>
+   <li>**After receiving the review, work on the required changes(if any) and submit again. Keep learning!!!</li>
+</ol>
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Knowledge of HTML, CSS, Linters, Git & GitHub flows and documenting a repo in a professional way
+
+### Setup
+A browser is required to be installed for view the contents and an Code editor to edit the code.
+
+### Install
+Clone the repository and use a browser to open the index.html to view the portfolio webpage
+
+👤 **Author1**
+
+- GitHub: [@Alabi12](https://github.com/azeezumoru)
+- Twitter: [@wolo_robert](https://twitter.com/azeez_umoru)
+- LinkedIn: [LinkedIn](linkedin.com/in/umoru-abdul-azeez-527921194)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check this site  https://github.com/azeezumoru/microverse-portfolio.git
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Good job 👍 adding a link to the MIT Licence into your README, but please remove _`Note:`_ from the LICENCE section of the README. Note is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it